docs/working-with-projects/working-with-images/repositories.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Repositories
+weight: 75
+---
+
+A repository is a collection of artifacts.  Since version v2.0, in addition to container images, Harbor can manage different kinds of artifacts that are bundled in OCI-compatible format, such as helm chart (requires helm v3), CNAB, etc.
+
+### List Repositories
+
+Click your project to enter the project detail page after successful logging in.  Click the "Repositories" tab to view the list of of repositories. 
+
+![list_repositories](../../../img/list-repositories.png)
+
+### Description of a repository
+
+Click the repository, then click the "Info" tab.  You can view the description of the project.  Users with project admin, master or developer role can click the "Edit" button to edit the description.  You can style the description via Markdown syntax.
+
+![edit_repository_description](../../../img/edit-repository-description.png)
+
+### List artifacts in a repository
+
+Click the "Artifacts" tab to view the list of artifacts in a repository.
+Each artifact is identified by its sha256 digest in the list of artifacts, and different types of artifacts can be distinguished by the icon on the left of the digest.  Hover your mouse on the icon you can see the name of the type.  
+
+By clicking the icon in the column **Pull Command**, the command to pull the artifact in the row of the icon will be copied to the clipboard.  
+The column **Annotations** in the grid shows the manifest annotations of the artifact, which are a set of key-value pairs.  More details about the annotations please refer to (https://github.com/opencontainers/image-spec/blob/master/annotations.md).
+The column **Push Time** in the grid shows the time each artifact is pushed to the registry.
+
+![list_artifacts](../../../img/list-artifacts.png)
+
+By clicking the search icon in the top right of the list of artifacts, you can different types of filter to filter the items in the artifact list.  You can choose to filter by type, tags, labels.  Particularly, if you choose to filter by tags, you choose to view only the tagged or untagged artifacts.
+
+![filter_artifacts](../../../img/filter-artifacts.png)
```

Comment:
should be ../../img? I can't view this png when I review through the file format.

Reply:
also confused but website is rendering from the this source file correctly

Reply:
yep, I have asked Jonas offline and waiting for his response... but 3 `../` seem the only way to make the website render correctly.

Reply:
ok, why I count as 2 ../? from the repositories.md file to access image in img directory, it seems only two level distance.

Reply:
ok, why I count as 2 ../? from the repositories.md file to access image in img directory, it seems only two level distance.

Reply:
It's because on the website, a file example such as:
https://goharbor.io/docs/1.10/administration/configure-authentication/db-auth/
links to image
https://goharbor.io/docs/1.10/img/db-auth.png

The file db-auth.md is treated as a directory itself, which is why you need 3 `../` instead of 2. I'll see if I can make this work somehow, but it's a common Hugo method of rendering the websites.

Reply:
Thanks Jonas.